src/services/mentees.js

```diff
@@ -622,14 +674,9 @@ module.exports = class MenteesHelper {
 	 * @param {Boolean} isAMentor 				- user mentor or not.
 	 * @returns {JSON} 							- List of filtered sessions
 	 */
-	static async filterSessionsBasedOnSaasPolicy(userId, isAMentor, tenantCode) {
+	static async filterSessionsBasedOnSaasPolicy(userId, isAMentor, tenantCode, orgCode) {
 		try {
-			const menteeExtension = await menteeQueries.getMenteeExtension(
-				userId,
-				['external_session_visibility', 'organization_id', 'is_mentor'],
-				false,
-				tenantCode
-			)
+			const menteeExtension = await cacheHelper.mentee.get(tenantCode, orgCode, userId)
```

Comment:
@sumanvpacewisdom we have thses columns in cache right ?
'external_session_visibility', 'organization_id', 'is_mentor'
We can try getting data from cache first if not found then we should get from db.